src/libs/ReportSecondaryActionUtils.ts

```diff
@@ -667,6 +670,11 @@ function getSecondaryReportActions({
         options.push(CONST.REPORT.SECONDARY_ACTIONS.CHANGE_WORKSPACE);
     }
 
+    // When report manager is not the policy admin and current user is policy admin, allow changing the approver
+    if (!isMemberPolicyAdmin(policy, getManagerAccountEmail(policy, report)) && isExpenseReportUtils(report) && isProcessingReportUtils(report) && isPolicyAdmin(policy)) {
```

Comment:
Oh hmm why should the current manager need to not be a policy admin? I think it's quite common for this scenario:

- Admin A is manager
- Admin A goes on vacation
- Admin C wants to assign Admin B to approve the report

So users could be all admins I think?